.github/workflows/storybook-pages.yml

```diff
@@ -0,0 +1,76 @@
+name: Storybook GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Should we restrict this to running only on pushes to master that have touched the js-pacakges dir?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
5040b0bff6a34c787a6dd13a88af200d1a612299